ipaserver/plugins/permission.py

```diff
@@ -279,7 +279,7 @@ class permission(baseldap.LDAPObject):
             label=_('Bind rule type'),
             doc=_('Bind rule type'),
             autofill=True,
-            values=(u'permission', u'all', u'anonymous'),
+            values=(u'permission', u'all', u'anonymous', 'self'),
```

Comment:
the other values explicitly marked as unicode, this one is not. Should this be fixed?

Reply:
It's Python 3, the ``u''`` prefix is just for show and doesn't affect the type.

Reply:
I left it out deliberately.

Reply:
It looks weird to have it all mixed on the same line. Either remove u'' or have it everywhere in that line.